geomstats/geometry/positive_lower_triangular_matrices.py

```diff
@@ -0,0 +1,359 @@
+"""The manifold of lower triangular matrices with positive diagonal elements"""
+
+import geomstats.backend as gs
+from geomstats.geometry.base import OpenSet
+from geomstats.geometry.matrices import Matrices
+from geomstats.geometry.riemannian_metric import RiemannianMetric
+from geomstats.geometry.lower_triangular_matrices import LowerTriangularMatrices
+
+
+class PositiveLowerTriangularMatrices(OpenSet):
```

Comment:
In the docstring, indicate that this space is also called the Cholesky space (so that other users who may have heard the term will relate as well)